src/App.vue

```diff
@@ -1,11 +1,21 @@
 <template>
   <div id="app">
+    <app-header />
     <router-view />
   </div>
 </template>
 
 <script>
+import Header from '@/components/Header.vue';
+import store from '@/store';
+
 export default {
-  name: 'App'
-}
+  name: 'App',
+  setup () {
+    store.commit('initStore')
+  },
+  components: {
+    'app-header': Header
```

Comment:
We need to establish name convention. I use for example just name of the component.

Reply:
done